README.md

```diff
@@ -1,29 +1,31 @@
 # Fomu Toolchain
 
-Fomu is an FPGA in your USB port.  This repository contains prebuilt versions of all the tools you will need to develop for Fomu.
+[Fomu](https://github.com/im-tomu/fomu-hardware) ([fomu.im](https://fomu.im)) is an FPGA in your USB port.  This repository gathers all the tools you will need to develop for Fomu, and provides them as prebuilt packages for GNU/Linux, Windows or macOS.
 
 ## Usage
 
 Download the [latest release](https://github.com/im-tomu/fomu-toolchain/releases/latest) for your platform and extract it somewhere on your disk.  Then set your PATH:
 
-* MacOS: `export PATH=[path-to-bin]:$PATH`
-* Linux: `export PATH=[path-to-bin]:$PATH`
-* Windows Powershell: `$ENV:PATH = "[path-to-bin];" + $ENV:PATH`
-* Windows cmd.exe: `PATH=[path-to-bin];%PATH%`
+* Shell (GNU/Linux, Cygwin/MSYS2/MINGW, MacOS...): `export PATH=[path-to-bin]:$PATH`
+* Powershell (Windows): `$ENV:PATH = "[path-to-bin];" + $ENV:PATH`
+* cmd.exe (Windows): `PATH=[path-to-bin];%PATH%`
 
 To confirm installation, run a command such as `nextpnr-ice40` or `yosys`.
 
 ## What's included
 
-This contains _almost_ everything you'll need to develop on Fomu:
-
-* **yosys** -- synthesis
-* **nextpnr-ice40** -- place-and-route
-* **dfu-util** -- upload bitstream to the FPGA
-* **python3** -- required for `nextpnr-ice40` and to build litex projects
-* **riscv-gcc** -- compile code for RISC-V CPUs, such as the Fomu softcore
-* **wishbone-tool** -- access the debug bus on Fomu
-
-Additionally, the macOS and Windows versions include `make`.
-
-It is strongly recommended that you install `git` to manage repositories and check out code, though it is not strictly necessary.
\ No newline at end of file
+Prebuilt packages contain _almost_ everything you'll need for developing software and/or hardware on Fomu:
+
+* [open-tool-forge/fpga-toolchain](https://github.com/open-tool-forge/fpga-toolchain):
+  * **yosys** -- synthesis
+  * **ghdl-yosys-plugin** -- VHDL frontend for Yosys
+  * **nextpnr** -- place-and-route
+  * **dfu-util** -- upload bitstream to the FPGA
+  * **python3** -- required for `nextpnr-ice40` and for building litex projects
```

Comment:
As per discussion in #16 I am deliberately not exposing a python3 interpreter as the lib/python3.8 dir is only intended for internal use by nextpnr.

If your workshop projects were relying on this you'll probably need to adjust accordingly.

Reply:
@edbording, you are correct, I overlooked that. I now added a note/clarification about it.